CHANGELOG.md

```diff
@@ -52,8 +60,20 @@ librdkafka v2.10.0 is a feature release:
 > The [KIP-848](https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol) consumer is currently in **Preview** and should not be used in production environments. Implementation is feature complete but contract could have minor changes before General Availability.
 
 
+ ## Upgrade considerations
+
+  Since this version threads for decommissioned or unavailable brokers are removed and
+  the client won't try to connect to one of them. Brokers and their threads
+  are added back when they appear in a Metadata RPC response again.
+  When no brokers are left or they're not reachable the client will start a
+  re-bootstrap sequence by default, to handle the case in a custom way set
+  the `metadata.recovery.strategy` to `none` and a fatal error will be
```

Comment:
There is no mention of Fatal error in the KIP. Why are we considering it as a Fatal Error?

Reply:
No, it shouldn't be given to the user in this KIP I just skip re-bootstrapping in that case.